### PR TITLE
ale 15

### PR DESCRIPTION
**Pull request by DonStella**
didn't get one question. some of the instructions are tricky to understand

### CODE REVIEW

File: 15-PB-conditionals-pt1/15ale.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+}
+// --> INCLUDES WRONG: doesn't make sure if it's at the beginning
+
+// Q5. 
+// used above numbers
+console.log("Q5");
+let mySum = x + y;
+
+if (80 <= mySum >= 50) {
+    console.log("65");
```
Comment (martl) [inline 15-PB-conditionals-pt1/15ale.js:63]: keep the correct data types !

File: 15-PB-conditionals-pt1/15ale.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+} else {
+    console.log("80");
```
Comment (martl) [inline 15-PB-conditionals-pt1/15ale.js:65]: keep the correct data types !

File: 15-PB-conditionals-pt1/15ale.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+} else {
+    console.log("need new numbers");
+}
+
+// Q10.
+console.log("Q10");
+let diff = int1 - 19; // -13
+
+if (diff > 19) {
+    console.log(diff * 2)
```
Comment (martl) [inline 15-PB-conditionals-pt1/15ale.js:116]: i think it should be printed times 3

Comment (DonStella) [reply 15-PB-conditionals-pt1/15ale.js:116]: it says print the difference double. 3 times is above.

File: 15-PB-conditionals-pt1/15ale.js (hunk 2 above)
Comment (DonStella) [reply 15-PB-conditionals-pt1/15ale.js:65]: what do u mean?

File: 15-PB-conditionals-pt1/15ale.js (hunk 1 above)
Comment (DonStella) [reply 15-PB-conditionals-pt1/15ale.js:63]: what do u mean?

**Pull request closed without merging**